l10n_th_address/__manifest__.py

```diff
@@ -32,6 +32,5 @@
     ],
     'installable': True,
     'auto_install': False,
+    'post_init_hook': '_update_data_noupdate',
```

Comment:
@pedrobaeza you mean "allowing to use..."?

Still, the check for author and license are configurable in pylint-odoo, so in our OCA configuration, these should be blocking errors no?